src/components/BeaconResults.vue

```diff
@@ -215,10 +215,24 @@ export default {
         this.response = [];
         this.notFound = true;
       }
+    },
+    setSearchToLocaStorage: function() {
+      if (localStorage.getItem("searches") == null) {
+        var searches = [];
+        var search = { url: window.location.href, date: new Date() };
```

Comment:
![image](https://user-images.githubusercontent.com/47524/125613555-c54df517-9b59-4bb8-964f-a4534ab23d0d.png)

The time stamp  in history looks different than what i have on my system :thinking: